alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/util/AlertingUtilsTests.kt

```diff
@@ -0,0 +1,58 @@
+package com.amazon.opendistroforelasticsearch.alerting.util
+
+import com.amazon.opendistroforelasticsearch.alerting.destination.message.BaseMessage
+import com.amazon.opendistroforelasticsearch.alerting.destination.message.CustomWebhookMessage
+import org.elasticsearch.test.ESTestCase
+import java.util.HashMap
+
+class AlertingUtilsTests : ESTestCase() {
+
+    private val denylistRanges = listOf(
+            "127.0.0.0/8",
+            "10.0.0.0/8",
+            "172.16.0.0/12",
+            "192.168.0.0/16",
+            "0.0.0.0/8"
+    )
+
+    private fun createMessageWithHost(host: String): BaseMessage {
```

Comment:
We should move these two private methods to the bottom so the the tests come first, to be consistent with the format in other tests, excluding the `denylistRanges` which can be at the top.